web/modules/custom/joinup_communities/semic/js/semic_community.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file
+ * Semic map renderer.
+ */
+
+L.custom = {
+  init: function (obj, params) {
+
+    // CUSTOM.
+    obj.style.minHeight = "400px";
+
+    // INIT MAP OBJECT.
+    var map = L.map(obj, {"center": [48, 9], "zoom": 4});
+
+    // TILES.
+    var tileLayer = L.wt.tileLayer().addTo(map);
+
+    // MARKERS.
+    var kml = document.getElementById('semic-map-kml').innerHTML.trim();
```

Comment:
This is very, very weird way to pass a variable from Drupal to JS. We should use `drupalSettings` for this purpose. See [Attaching configurable JavaScript](https://www.drupal.org/docs/8/creating-custom-modules/adding-stylesheets-css-and-javascript-js-to-a-drupal-8-module#configurable).